spec/lib/os_places_api/client_spec.rb

```diff
@@ -270,4 +233,23 @@
       end
     end
   end
+
+  describe "#update_postcode" do
+    context "the postcode is outdated in the database" do
```

Comment:
We can remove this `context` line (the `update_postcode` method will attempt to update the postcode regardless of whether or not it is out of date).